Ginger/Ginger/Extensions/ExtensionsMethods.cs

```diff
@@ -338,6 +338,25 @@ public static void SetContent(this Frame Frame, Page Page)
         // Validations
         // ------------------------------------------------------------
 
+        public static void RemoveValidations(this FrameworkElement frameworkElement)
+        {
+            BindingExpression bd = null;
+            
+            if(frameworkElement is ucAgentControl)
+            {
+                bd = frameworkElement.GetBindingExpression(ucAgentControl.SelectedAgentProperty);
+            }
+            else if(frameworkElement is ComboBox)
+            {
+                bd = frameworkElement.GetBindingExpression(ComboBox.SelectedValueProperty);
+            }
+
```

Comment:
change the method to be generic, get the dependency Property (like: SelectedAgentProperty, SelectedValueProperty) also as parameter instead of those If-Else

Reply:
Fixed